app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+<% if current_page?(new_tourist_session_path || new_agency_session_path) %>
+  <%= render "devise/shared/links" %>
+<% end %>
+
```

Comment:
Can you please render the design I created? Thanks mate!

Reply:
I will add in the next PR